pkg/api/validate.go

```diff
@@ -98,7 +98,17 @@ func (p *Properties) validateVMProfile(vmconf VMConfigurator) error {
 	if e := validateOSDisk(vm.OSDisk); e != nil {
 		return e
 	}
-
+	if(vm.SecurityProfile == nil) {
+		vm.SecurityProfile = &SecurityProfile{ "true",false}
```

Comment:
How about setting both secureboot and vtpm "true" by default?

Reply:
Sure, resolved this in the new iteration.